apps/overige-objecten-api/src/utils/getVacData.ts

```diff
@@ -0,0 +1,52 @@
+import kebabCase from 'lodash.kebabcase';
+import { concatenateFieldValues } from './concatenateFieldValues';
+import type { Antwoord, DataVacItem, VACSData } from '../strapi-product-type';
+
+interface GetVacDataProps {
+  data: VACSData;
+  vacSchemaURL: string;
+  serverURL: string;
+}
+interface MapVAcItemProps extends Pick<GetVacDataProps, 'serverURL' | 'vacSchemaURL'> {
+  item: DataVacItem;
+}
+
+const getAntwoord = (antwoord: Antwoord[]): string => (Array.isArray(antwoord) ? concatenateFieldValues(antwoord) : '');
+
+const createVacRecord = (item: DataVacItem, vacUrl: string, antwoord: string) => {
```

Comment:
Hoe weten we of de return value compleet is? Is hier geen typescript interface voor?